src/hardware/pounder/mod.rs

```diff
@@ -350,19 +381,33 @@ impl PounderDevices {
         // Configure power-on-default state for pounder. All LEDs are off, on-board oscillator
         // selected and enabled, attenuators out of reset. Note that testing indicates the
         // output state needs to be set first to properly update the output registers.
-        devices
-            .mcp23017
-            .all_pin_mode(mcp23017::PinMode::OUTPUT)
-            .map_err(|_| Error::I2c)?;
-        devices
-            .mcp23017
-            .write_gpio(mcp23017::Port::GPIOA, 0x00)
-            .map_err(|_| Error::I2c)?;
-        devices
-            .mcp23017
-            .write_gpio(mcp23017::Port::GPIOB, 0x2F)
-            .map_err(|_| Error::I2c)?;
-
+        for pin in [
+            GpioPin::AttLe1,
```

Comment:
This seems like an easy place to forget to add one of the `GpioPin` members. Why not use https://crates.io/crates/iter-enum?

Reply:
That's old/dead I think.
https://crates.io/crates/enum-iterator looks good.